lib/build.js

```diff
@@ -21,26 +21,34 @@ module.exports = {
       enum: [ 'Toggle', 'Keep Visible', 'Show on Error' ],
       order: 1
     },
+    panelOrientation: {
+      title: 'Panel Orientation',
+      description: 'Set where the build panel should be.',
+      type: 'string',
+      default: 'Bottom',
+      enum: [ 'Top', 'Bottom', 'Left', 'Right' ],
```

Comment:
I understand. My original thought it was to have an option for the people that liked using the split function for their code between left and right. However, it looks like even if they had a split it would create a third window in the split, and you're right. To read the error _and_ look at the now squished windowed code to fix it an error, is laughably awkward.

I'll make try and make the changes to get it to orientate to the top/bottom if you still want it. Though that task is so trivial you could just do it yourself and save yourself the pull. Sorry for the time waster man.